setup.py

```diff
@@ -67,6 +67,7 @@
         'urllib3[secure] < 1.25 ; python_version < "3.5"',
         'urllib3[secure] < 2 ; python_version >= "3.5"',
         "wrapt>=1.10,<2.0",
+        "requests",
```

Comment:
If we are trying to limit our dependencies, we can use `urllib3`, but `requests` makes things much cleaner.